readme.md

```diff
@@ -217,30 +219,44 @@ Contains libraries that enable interactions with the Device Proviosining service
 
 Contains libraries that provide modeling and JSON serialization capabilities on top of the raw messaging library.
 
+# Releases
 
-# Long Term Support
+The C SDK offers releases for new features, critical bug fixes, and Long Term Support (LTS). General bug fixes will not receive a separate release. Versioning follows [semantic versioning](https://semver.org/), `x.y.z.` or `major.minor.patch`. Any time the version is updated, it will be tagged `x.y.z`.
```

Comment:
```suggestion
The C SDK offers releases for new features, critical bug fixes, and Long Term Support (LTS). General bug fixes will not receive a separate release and are contained in the LTS release. Versioning follows [semantic versioning](https://semver.org/), `x.y.z` or `major.minor.patch`. Any time the version is updated, it will be tagged `x.y.z`.
```